libs/api/domains/auth/src/lib/resolvers/domain.resolver.ts

```diff
@@ -28,4 +34,18 @@ export class DomainResolver {
   ): Promise<Domain[]> {
     return this.domain.getDomains(user, input)
   }
+
+  @ResolveField('organisationLogoUrl', () => String, { nullable: true })
+  async resolveOrganisationLogoKey(
+    @Loader(OrganizationLogoLoader)
+    organizationLogoLoader: OrganizationLogoDataLoader,
+    @Parent() domain: Domain,
+    @Args('lang', { type: () => String, nullable: true, defaultValue: 'is' })
+    lang: string,
```

Comment:
Organisation logos should not be translated. Can get rid of the lang argument and the object below. Just pass in the logo key directly.